lib/elixir/src/elixir_module.erl

```diff
@@ -226,11 +231,12 @@ build(Line, File, Module, Lexical) ->
 
 eval_form(Line, Module, Data, Block, Vars, E) ->
   {Value, EE} = elixir_compiler:eval_forms(Block, Vars, E),
-  elixir_overridable:store_pending(Module),
+  OverridableDefinitions = elixir_overridable:store_pending(Module),
   EV = elixir_env:linify({Line, reset_env(EE)}),
   EC = eval_callbacks(Line, Data, before_compile, [EV], EV),
-  elixir_overridable:store_pending(Module),
-  {Value, EC}.
+  OverriddenDefinitions = elixir_overridable:store_pending(Module),
+  OverridablePairsNotOverridden = OverridableDefinitions -- OverriddenDefinitions,
```

Comment:
Are you absolutely sure? Since in my testing it seems like the prior is always a superset of the latter, so then we end up with duplicates.

There is little to no naming of these things in elixir_module.erl so I am making guesses as to what they actually mean.